### PR TITLE
[NEW] upload media to order

### PR DESCRIPTION
**Pull request by WLsupport**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,6 @@ confdoc.ser
 .buildpath
 .project
 .settings/
+
+.idea
+vendor
```
Comment (aimeos) [inline .gitignore:24]: Can you please remove those lines as they are not related to the feature itself?

Comment (WLsupport) [reply .gitignore]: the lines are removed
